marketplace_builder/views/pages/tutorials/users/create-user-property.liquid

```diff
@@ -0,0 +1,39 @@
+---
+converter: markdown
+metadata:
+  title: Creating a property for all Users
+  description: This guide will help you create a property for all Users for your platformOS site.
+slug: tutorials/users/creating-user-property
+searchable: true
+---
+
+This guide will help you create a property for all Users for your platformOS site.
+
+## Requirements
+
+To follow the steps in this tutorial, you should be familiar with the required directory structure for your codebase, and understand the concept of users.
+
+* [Codebase](/how-platformos-works/platformos-workflow)
+* [Users](/tutorials/users/users)
+
+## Steps
+
+Creating a property for all Users is a one-step process:
+
+<div data-autosteps></div>
+
+### Step 1: Create yml file
```

Comment:
I got lost somewhere between step 1 and step 1 ;)

Reply:
try again starting from the beginning :P